tasks/electron-react-ts/README.md

```diff
@@ -17,3 +17,38 @@ If the Bluetooth Low Energy P2P transport is enabled, the application might cras
 ## Bluetooth Low Energy on Linux
 
 Please refer to [the docs](https://docs.ditto.live/raspberrypi/installation) for instructions on how to enable Bluetooth Low Energy on Linux.
+
+## Windows integration
+
+1. Push Win Key and type "Services", locate those services, start them and set Start type to Automatic.
```

Comment:
```suggestion
1. Push Win Key and type "Services", locate each of the below services, start them and set Start type to Automatic.
```

I'm also not familiar enough with the process on Windows here, so wanted to double check the order of `start them and set Start type to Automatic` doesn't matter. Intuitively, I would have thought it'd be reversed (setting start type before starting), but I rarely use Windows so defer to others.

Reply:
A customer reported that this step wasn't required so we might be able to just remove this section altogether